test-cases/longevity/longevity-alternator-200GB-48h.yaml

```diff
@@ -0,0 +1,33 @@
+test_duration: 180
+prepare_write_cmd: [
+  "bin/ycsb load dynamodb -P workloads/workloada -threads 250 -p recordcount=200200300 -p fieldcount=1 -p fieldlength=1024 -p dataintegrity=true -p hdrhistogram.summary.addstatus=true", ]
+stress_cmd: [
+  "bin/ycsb run  dynamodb -P workloads/workloada -threads 250 -p recordcount=200200300 -p fieldcount=1 -p fieldlength=1024 -p operationcount=400400600 -p dataintegrity=true -p hdrhistogram.summary.addstatus=true", ]
+
```

Comment:
What is the load in the cluster in terms of CPU when this stress run?

Reply:
we don't know we didn't yet run this.